### DISCUSSION TITLE
GOSSIP DNS

### DISCUSSION THREAD

**Discussion by Th4n05** [eddycharly/terraform-provider-kops#892] [Q&A] [answered]
Hi, 
I'm trying to run kops with Gossip DNS (k8s.local) configurations:
The execution returns:
`
│ Error: error running tasks: deadline exceeded executing task ClassicLoadBalancer/*.k8s.local. Example error: Field is required: Subnets │.`
`│   with kops_cluster_updater.updater, 
│   on main.tf line 90, in resource "kops_cluster_updater" "updater":      
│   90: resource "kops_cluster_updater" "updater" {`.     
Do you have any suggestion to resolve it ?

**Comment by Th4n05** [comment, marked as answer]
I resolved it by defining masters,nodes,dns topology public instead of private

**Reply by eddycharly** [reply]
Awesome, thanks for letting me know !
